### DISCUSSION TITLE
No pybricks on fresh installation

### DISCUSSION THREAD

**Discussion by sivko2** [pybricks/support#637] [Q&A]
Hi!

If I launch the app:
robot@ev3dev:~$ python3 vozilo.py
Traceback (most recent call last):
  File "vozilo.py", line 2, in <module>
    from pybricks import ev3brick as brick
ImportError: No module named 'pybricks'

The test app is quite simple:
robot@ev3dev:~$ more vozilo.py
#!/usr/bin/env pybricks-micropython
from pybricks import ev3brick as brick

print("Hello")
robot@ev3dev:~$

Thank you very much in advance.

Cheers, Roman

**Comment by sivko2** [comment]
I installed PIP and used pip3 to install pybricks.

The python app runs OK but EV3 is not moving at all w/o any errors.

**Comment by laurensvalk** [comment]
When you run a script like this:

```
 python3 vozilo.py
```

It will be run with `python3`, which is standard Python.

See [these instructions](https://pybricks.com/ev3-micropython/startrun.html) to run Pybricks programs.

If you really want to run Pybricks programs from the terminal, you can do:

```
brickrun -r -- pybricks-micropython /home/robot/vozilo.py
```

`pybricks-micropython` is the MicroPython executable, while `brickrun` is an ev3dev program that takes care of clearing the screen and stopping the motors when the program stops.

**Reply by sivko2** [reply]
Now I get:
ImportError: no module named ‘flask’

**Reply by dlech** [reply]
MicroPython is not standard Python, so Python packages like `flask` are not available. You can either use Python 3 or Pybricks MicroPython, but you can't use both at the same time. But you could write two programs, one in Python 3 and one in Pybricks MicroPython that communicate with each other if you need features from both.
